modules/ROOT/pages/monitor-api-conformance.adoc

```diff
@@ -10,6 +10,8 @@ You can view the latest status information in the console.
 
 include::_partials/api-gov-console-scap-callouts.adoc[]
 
+NOTE: Only API Governance and organization administrators can create, edit, and delete profiles.
```

Comment:
Consider making this a partial?

Reply:
There are only two occurrences, so I made a judgement call. I can, though.

Reply:
Deferring to another release